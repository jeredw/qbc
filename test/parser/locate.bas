locate
locate 20
locate 20, 2, 0, 31
locate 20, 2, 0
locate 20, 2,  , 31
locate 20, 2
locate 20, , 0, 31
locate 20, , 0
locate 20, , , 31
