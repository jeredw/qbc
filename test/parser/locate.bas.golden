(program 
  (statement 
    (locate_statement locate)) \n 
  (statement 
    (locate_statement locate 
      (expr 
        (literal 20)))) \n 
  (statement 
    (locate_statement locate 
      (expr 
        (literal 20)) , 
      (expr 
        (literal 2)) , 
      (expr 
        (literal 0)) , 
      (expr 
        (literal 31)))) \n 
  (statement 
    (locate_statement locate 
      (expr 
        (literal 20)) , 
      (expr 
        (literal 2)) , 
      (expr 
        (literal 0)))) \n 
  (statement 
    (locate_statement locate 
      (expr 
        (literal 20)) , 
      (expr 
        (literal 2)) , , 
      (expr 
        (literal 31)))) \n 
  (statement 
    (locate_statement locate 
      (expr 
        (literal 20)) , 
      (expr 
        (literal 2)))) \n 
  (statement 
    (locate_statement locate 
      (expr 
        (literal 20)) , , 
      (expr 
        (literal 0)) , 
      (expr 
        (literal 31)))) \n 
  (statement 
    (locate_statement locate 
      (expr 
        (literal 20)) , , 
      (expr 
        (literal 0)))) \n 
  (statement 
    (locate_statement locate 
      (expr 
        (literal 20)) , , , 
      (expr 
        (literal 31)))) \n <EOF>)
